lib/linter-rust.coffee

```diff
@@ -17,6 +17,9 @@ class LinterRust
       results = []
       file = @initCmd do textEditor.getPath
       curDir = path.dirname file
+      PATH = path.dirname @cmd[0]
+      options = @makeOpts(PATH)
+      options.cwd = curDir
       @cmd.push file
       command = @cmd[0]
       options = {cwd: curDir}
```

Comment:
I think that here the variable ```options``` will be overwritten and all previous steps do not make sense.